### PR TITLE
Add SSL_CERT_DIR & SSL_CERT_FILE to trusted certs

### PR DESCRIPTION
**Pull request by ransombriggs**

### CODE REVIEW

**Comment by ransombriggs** [comment]
@dickeyxxx review me?

**Comment by ransombriggs** [comment]
@dickeyxxx my bad, will fix the tests

File: lib/request.js [hunk 1]
```diff
@@ -59,6 +66,34 @@ function Request(options, callback) {
...
+function getCerts(debug) {
+  var filenames = sslCertFile().concat(sslCertDir());
+
+  if (filenames.length > 0 && debug) {
+    console.error('Adding the following trusted certificate authorities');
+  }
+
+  return filenames.map(function(filename) {
+    if (debug) {
+      console.error('  ' + filename);
```
Comment (dickeyxxx) [inline lib/request.js]: why not use `hush`?

Comment (ransombriggs) [inline lib/request.js]: @dickeyxxx because hush is part of heroku-cli-util and did not want to introduce a dependency

**Comment by dickeyxxx** [comment]
lgtm

**Pull request merged**